js/main.js

```diff
@@ -1,125 +1,138 @@
+    resultSection.appendChild(resultDataList);
   }
-  resultSection.appendChild(recipeList);
-  resultSection.appendChild(recipeListTwo);
 }
 
-/*
-results: Array(12), offset: 0, number: 12, totalResults: 55}
-number: 12
-offset: 0
-results: Array(12)
-0:
-aggregateLikes: 17
-analyzedInstructions: [{…}]
-cheap: false
-cookingMinutes: -1
-creditsText: "Jen West"
-cuisines: []
-dairyFree: true
-diets: (6) ['gluten free', 'dairy free', 'paleolithic', 'lacto ovo vegetarian', 'primal', 'vegan']
-dishTypes: ['salad']
-gaps: "no"
-glutenFree: true
-healthScore: 95
-id: 715540
-image: "https://spoonacular.com/recipeImages/715540-312x231.jpg"
-imageType: "jpg"
-lowFodmap: false
-nutrition: {nutrients: Array(29), properties: Array(2), flavonoids: Array(26), ingredients: Array(5), caloricBreakdown: {…}, …}
-occasions: (2) ['4th of july', 'summer']
-openLicense: -1
-preparationMinutes: -1
-pricePerServing: 175.37
-readyInMinutes: 45
-servings: 1
-sourceName: "Pink When"
-sourceUrl: "http://www.pinkwhen.com/summer-berry-salad/"
-spoonacularSourceUrl: "https://spoonacular.com/summer-berry-salad-715540"
-summary: "Summer Berry Salad might be just the side dish you are searching for. Watching your figure? This caveman, gluten free, primal, and vegan recipe has <b>98 calories</b>, <b>2g of protein</b>, and <b>1g of fat</b> per serving. This recipe serves 1 and costs $1.75 per serving. 17 people have tried and liked this recipe. If you have apples, blackberries, romaine lettuce, and a few other ingredients on hand, you can make it. It can be enjoyed any time, but it is especially good for <b>The Fourth Of July</b>. From preparation to the plate, this recipe takes approximately <b>45 minutes</b>. All things considered, we decided this recipe <b>deserves a spoonacular score of 98%</b>. This score is outstanding. Try <a href=\"https://spoonacular.com/recipes/berry-summer-salad-50848\">Berry Summer Salad</a>, <a href=\"https://spoonacular.com/recipes/summer-berry-salad-485592\">Summer Berry Salad</a>, and <a href=\"https://spoonacular.com/recipes/chicken-berry-summer-salad-769303\">Chicken & Berry Summer Salad</a> for similar recipes."
-sustainable: false
-title: "Summer Berry Salad"
-vegan: true
-vegetarian: true
-veryHealthy: true
-veryPopular: false
-weightWatcherSmartPoints: 0
-*/
+// function definition with 6 parameters
+// create an element with the first parameter
+// create and update attribute values with remaining parameters
+// return the element value
+function singleCard(type, className, id, bgImage, title, duration) {
+  const elementType = document.createElement(type);
+  if (className) {
+    elementType.className = className;
+  }
+  if (id) {
+    elementType.id = id;
+  }
+  if (bgImage) {
+    elementType.style.backgroundImage = ('url(' + bgImage + ')');
+  }
+  if (title) {
+    elementType.textContent = title;
+  }
+  if (duration) {
+    elementType.textContent = duration;
+  }
+  return elementType;
+}
```

Comment:
For a utility function like this that takes more than three or so parameters, you may want to combine the optional ones into a single `options` object parameter with corresponding key names (`options.className`, `options.id`, etc.). Also, what's the difference between `title` and `duration` here?